### PR TITLE
VB-1219: Remove top pagination from prisoner search results

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: server/views/pages/search/prisonerResults.njk [hunk 1]
```diff
@@ -36,27 +36,25 @@
...
-                {
-                  text: "Prison number"
-                },
-                {
-                  text: "Date of birth"
-                }
-            ],
-            rows: results
-        }) }}
+      {{ govukTable({
```
Comment (psoleckimoj) [inline server/views/pages/search/prisonerResults.njk:39]: Just un(?)dented.

**Review by tpmcgowan — DISMISSED** [review]
LGTM - only comment is that there are a few other places where pagination is used (from having searched code for `mojPagination`) so wondering if these should be treated similarly for consistency?

**Comment by psoleckimoj** [comment]
@tpmcgowan ticket updated to do all, will sort after this meeting I'm in

**Comment by psoleckimoj** [comment]
@tpmcgowan @hutcheonb removed top paging from all results pages as per Phoebe B's request.

**Review by tpmcgowan — APPROVED** [review]
LGTM!

**Pull request merged**